test/utils/database.ts

```diff
@@ -0,0 +1,33 @@
+import { Container } from 'typedi';
+import { Connection, createConnection, useContainer } from 'typeorm';
+
+import { env } from '../../src/env';
+
+declare type LoggerOptions = boolean | 'all' | Array<('query' | 'schema' | 'error' | 'warn' | 'info' | 'log' | 'migration')>;
+
+export const createDatabaseConnection = async (): Promise<Connection> => {
+    useContainer(Container);
+    const connection = await createConnection({
+        type: env.db.type as any, // See createConnection options for valid types
+        database: env.db.database,
+        logging: env.db.logging as LoggerOptions,
+        entities: env.app.dirs.entities,
+        migrations: env.app.dirs.migrations,
+    });
+    console.log('Rupank', connection);
```

Comment:
Use Logger class

Reply:
@VivekFitkariwala Removed the log statement as it is not required at the moment.